cv_scan_ai/src/tools/text_reviewer/config.py

```diff
@@ -0,0 +1,7 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv('../.env')
+
+openai_api_key = os.getenv('OPENAI_API_KEY')
+openai_model = os.getenv('OPENAI_MODEL', "gpt-3.5-turbo")
```

Comment:
Why `gpt-3.5-turbo`?

Reply:
Second parameter is default value, you have to specify the value in the environment only if you want to change it.

Reply:
ok,lets leave it configurable in that way. Though I think it is worth trying to play with other models during prompt engineering task to see if any other model gives better result for our specific task.